lib/game.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require_relative 'board.rb'
+require_relative 'player.rb'
+
+class Game
+  def initialize(player_one, player_two, board)
+    @player_one = player_one
+    @player_two = player_two
+    @board = board
+  end
+
+  def switch_player(active_player)
+    if active_player == @player_one
+      active_player = @player_two
+    elsif active_player == @player_two
+      active_player = @player_one
+    end
```

Comment:
The switch here is between two items, this should work 
 `active_player = active_player == @player_one ? @player_two : @player_two`

Reply:
your ternary operator is saying
if true => player_two
if false => player_two

Was this on purpose?